libp2p/daemon.ss

```diff
@@ -14,6 +14,8 @@
 (def current-libp2p-daemon
   (make-parameter #f))
 
+;; Starts a new libp2p-daemon only if there is no existing current-libp2p-daemon,
+;; and if so sets the current-libp2p-daemon to the new one.
```

Comment:
Maybe that should be renamed `ensure-libp2p-daemon` ?
Or, if we want backward-compatibility, a `force:` or `new:` boolean flag?